packages/orchestration/src/examples/swapExample.contract.js

```diff
@@ -105,15 +105,15 @@ export const start = async (zcf, privateArgs, baggage) => {
     async (/** @type {Orchestrator} */ orch, { zcf }, seat, offerArgs) => {
       const { give } = seat.getProposal();
 
-      const celestia = await orch.getChain('celestia');
+      const omni = await orch.getChain('omniflixhub');
```

Comment:
@dtribble I think you presented this celestia example publicly and the choice of chain was not arbitrary, so this is in some sense product API surface. But we're using more real data now, and there is no connection to celestia yet. Be advised: we're changing this example to one of [Agoric's current peer zones](https://mapofzones.com/zones/agoric-3/peers?columnKey=ibcVolumeIn&period=24h).

cc @mitdralla